en/aos-started.md

```diff
@@ -412,6 +414,34 @@ android {
 ```
 Caution: Huawei App Gallery must be installed on your device to make payment normally.
 
+#### MyCard
+
+* To integrate with MyCard payment, you must use GamebaseMyCardApplication. Add the following to the AndroidManifest.xml.
+
+```xml
+<application
+    android:name="com.toast.android.gamebase.purchase.mycard.GamebaseMyCardApplication"
+  ...>
+  ...
+</application>
+
+```
+
+* If you define and use your own application, inherit from GamebaseMyCardApplication.
```

Comment:
안녕하세요
'application' 이 일반적으로 '앱'으로 통용되는 단어와 같아서 application -> application class 로 변경하고 뒤에 "by extending Application class" 을 추가하는게 좀더 명확할것 같은데 검토해주실수 있으실까요?

If you define and use your own application class by extending Application class, inherit from GamebaseMyCardApplication.

Reply:
안녕하세요. 제안해 주신 문장이 의미전달이 더욱 명확할 것 같아 반영하겠습니다. 뒤에 'instead of Application class'는 'by extending ~'과 의미중복되어 생략하여 아래와 같이 수정하였습니다. 

If you define and use your application class by extending Applcation class, inherit from GamebaseMyCardApplication.

Reply:
감사합니다!

Reply:
감사합니다!